src/modules/test/main.cpp

```diff
@@ -96,6 +96,8 @@ int main(int argc, char **argv)
         PrintUsage();
     }
 
+    // TODO: add fulllogging on/off?
+    SetFullLogging(true);
```

Comment:
We discussed to not add OSConfig logging here. This  test binary does not have a OSConfig log. Remove this (lines 99 and 100)